rskj-core/src/test/java/org/ethereum/rpc/parameters/BlockRefParamTest.java

```diff
@@ -0,0 +1,68 @@
+package org.ethereum.rpc.parameters;
+
+import org.ethereum.rpc.BlockRef;
+import org.ethereum.rpc.exception.RskJsonRpcRequestException;
+import org.junit.jupiter.api.Test;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+public class BlockRefParamTest {
```

Comment:
These are all the example cases we should be supported by BlockRefParam class
- "earliest"
- "0x0"
- { "blockNumber": "0x0" }
- { "blockHash": "0xd4e56740f876aef8c010b86a40d5f56745a118d0906a34e69aec8c0db1cb8fa3" } (hash of the genesis block on the Ethereum main chain)
- { "blockHash": "0xd4e56740f876aef8c010b86a40d5f56745a118d0906a34e69aec8c0db1cb8fa3", "requireCanonical": true }
- { "blockHash": "0xd4e56740f876aef8c010b86a40d5f56745a118d0906a34e69aec8c0db1cb8fa3", "requireCanonical": false }

And I think at least we should test them to be sure that we are supporting them.